db/seeds.rb

```diff
@@ -135,10 +137,11 @@
 
 p 'assigning worksheets to work groups'
 
-work_groups.each_with_index do |work_group, idx|
-  template = idx == WorkGroup.count - 1 ? WorksheetTemplate.last : WorksheetTemplate.first
+work_groups = WorkGroup.all
+work_groups.each_with_index do |work_group, index|
+  template = index % 2 == 0 ? WorksheetTemplate.last : WorksheetTemplate.first
```

Comment:
like this different work groups from the same class will have different worksheets, when in reality each workgroup from a specific class should we working on the same worksheet